README.md

```diff
@@ -36,14 +36,15 @@ Currently this means:
 * Python 3.7+
 * Qt 5.12-5.15, 6.1
 * Required
-  * PyQt5, PyQt6, PySide2 or PySide6
-  * `numpy` 1.17+
+  * [PyQt5](https://www.riverbankcomputing.com/software/pyqt/), [PyQt6](https://www.riverbankcomputing.com/software/pyqt/), [PySide2](https://wiki.qt.io/Qt_for_Python) or [PySide6](https://wiki.qt.io/Qt_for_Python)
+  * [`numpy`](https://github.com/numpy/numpy) 1.17+
 * Optional
-  * `scipy` for image processing
-  * `pyopengl` for 3D graphics
-    * `pyopengl` on macOS Big Sur only works with python 3.9.1+
-  * `hdf5` for large hdf5 binary format support
-  * `colorcet` for supplemental colormaps
+  * [`scipy`](https://github.com/scipy/scipy) for image processing
+  * [`pyopengl`](https://github.com/mcfletch/pyopengl) for 3D graphics
+    * [`pyopengl`](https://github.com/mcfletch/pyopengl) on macOS Big Sur only works with python 3.9.1+
+  * [`h5py`](https://github.com/h5py/h5py) for large hdf5 binary format support
+  * [`colorcet`](https://github.com/holoviz/colorcet) for supplemental colormaps
+  * [`matplotlib`](https://github.com/matplotlib/matplotlib) for supplemental colormaps
```

Comment:
Is it worth also mentioning some exporters can be converted to matplotlib with this dependency?

Reply:
I would say yes, definitively.
Is there other optional stuffs to add in the README (fr matplotlib or others)?

Reply:
> I would say yes, definitively.
> Is there other optional stuffs to add in the README (fr matplotlib or others)?

Perhaps with discussing the speedups `numba` provide (I believe right now it's limited to `ImageItem.setData()` for repeated calls).  We've talked about adding some other libraries for optional support (pandas); but we have a ways to go before we get there.

If you don't think any of the existing sections would fit to discuss the optional dependent specifics, feel free to create a separate section.

Reply:
Also, is PyOpenGL-accelerate worth mentioning alongside PyOpenGL? I don't use either at all; I just know is that they are often referred to together.

Reply:
>Perhaps with discussing the speedups numba provide (I believe right now it's limited to ImageItem.setData() for repeated calls).

Changing the image raw data is one way to trigger rescaling, yes. The other way is by changing the levels of the colorbar. For example, in #1512, changing the levels interactively through the HistogramLUTItem will call rescale several times.